x509-identity-mgmt/ejbca/bin/after-deployed-pre.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+##################################################################
+#                                                                #
+# Copyright (c) 2020 Dojot IoT Platform                          #
+#                                                                #
+# This software is free software; you can redistribute it and/or #
+# modify it under the terms of the GNU Lesser General Public     #
+# License as published by the Free Software Foundation; either   #
+# version 2.1 of the License, or any later version.              #
+#                                                                #
+# See terms of license at gnu.org.                               #
+#                                                                #
+##################################################################
```

Comment:
Make sense describe license terms inside project files ?, It is not easier create a LICENSE file to the whole component (for example https://github.com/dojot/iotagent-leshan/blob/development/LICENSE)

Reply:
Good question! I believe your placement is correct. For Bash scripts, they are included together from the EJBCA scripts, and how it defined the license in the header file, I just kept the same approach. We will certainly review this license issue in the future.

Reply:
Ok, I got it.